### ISSUE TITLE
Increase code coverages 

### ISSUE THREAD

**Issue by nazrulworld** [mmabey/fhirdatetime#2]
We should have aim to make 100% code coverages.

**Comment by mmabey** [comment]
I agree that it would be better to have higher coverage. But if you take a closer look at the coverage details (see image below), you'll see that the reason coverage is so low is because of the `_datetime` file, which is 99% code that I copied directly from the standard library.

![image](https://user-images.githubusercontent.com/1606251/116733592-83c19080-a9a9-11eb-9686-fda8f365fda1.png)

I didn't put the time into writing more tests prior to now because (1) it feels silly to test code from the standard library too rigorously and (2) I had time constraints so I prioritized testing the new code I had written.

I could very easily boost the coverage score just by excluding the `_datetime` file, but that feels like it makes the coverage score deceptive. What are your thoughts?
